### DISCUSSION TITLE
Expo Goは同一ネットワーク上にある端末のみ使用できることを明記する

### DISCUSSION THREAD

**Discussion by takei0w0** [ws-4020/mobile-app-crib-notes#218] [Ideas]
Getting Started - アプリの実行にて、Expo Goは別ネットワーク上の端末で利用できないことが書かれていない。
別ネットワークで作業している学習者はiOSシミュレーターやAndroidシミュレーターを使うよう明記する。

**Comment by creature-water-valley** [comment]
一応以下の記載はある。
>expo start を実行するとExpoの開発者ツールとMetro Bundlerが起動します。 開発者ツールに記載されているQRコードを読み取ることで、Expo Goは同一無線ネットワーク内のMetro Bundlerに接続するよう設定されます。そしてMetro Bundlerを通してJava Scriptのソースを読み込み、実行します。

https://ws-4020.github.io/mobile-app-crib-notes/react-native/learn/getting-started/launch-created-app
